lib/index.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const {config} = bedrock;
+const si = require('systeminformation');
+require('bedrock-stats');
+
+require('./config');
+const cfg = config['stats-monitor-os'];
+
+bedrock.events.on('bedrock-stats.report', async ({monitors}) => {
+  const os = monitors.os = {};
+  for(const metric in cfg.metrics) {
+    os[metric] = await si[metric]();
```

Comment:
Could run these all at once.  I imagine one of the [promise tools](https://github.com/sindresorhus/promise-fun) would help with this pattern.

Reply:
pProps used FTW.